en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -59,7 +87,19 @@ The following code snippet contains sample configuration of the parameter file.
               production:
                   url: 'https://test.wso2.com'
                   config:
-                      retryTimeOut: $RETRY
+                    factor: 3
```

Comment:
```suggestion
                      factor: 3
```
Let's indent all the fields like this? (the earlier _retryTimeOut: $RETRY_ had correct indentation I guess)